src/WpBookingsFrontUi.php

```diff
@@ -1,144 +1,256 @@
+                        'sessionSelectorTemplate'        => $c->get('eddbk_front_session_selector_template'),
+                        'wizardServiceStepTemplate'      => $c->get('eddbk_front_service_step_template'),
+                        'wizardSessionStepTemplate'      => $c->get('eddbk_front_session_step_template'),
+                        'wizardConfirmationStepTemplate' => $c->get('eddbk_front_confirmation_step_template'),
+                    ]);
+                },
+
+                'eddbk_wizard_assets_urls_map' => function (ContainerInterface $c) {
+                    $containerFactory = $this->_getContainerFactory();
+                    $assetsUrlsMap = require_once $c->get('bookings_front_ui/assets_urls_map_path');
+
+                    return $containerFactory->make([
+                        ContainerFactoryInterface::K_DATA => $assetsUrlsMap,
+                    ]);
+                },
+
+                'eddbk_wizard_assets' => function (ContainerInterface $c) {
+                    return $c->get('bookings_front_ui/assets');
+                },
+
+                /*
+                 * Handles assets enqueuing for application.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_wizard_enqueue_assets_handler' => function (ContainerInterface $c) {
+                    return new AssetsEnqueueHandler($c->get('eddbk_wizard_assets_urls_map'), $c->get('eddbk_wizard_assets'));
+                },
+
+                /*
+                 * Handles state output for client application.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_wizard_enqueue_app_state_handler' => function (ContainerInterface $c) {
+                    return new StateEnqueueHandler(
+                        $c->get('bookings_front_ui/application_selector'),
+                        $c->get('bookings_front_ui/api_endpoint_urls'),
+                        $c->get('bookings_front_ui/booking_data_map'),
+                        $c->get('bookings_front_ui/initial_booking_transition'),
+                        $c->get('bookings_front_ui/formats/datetime')
+                    );
+                },
+
+                /*
+                 * Handles for outputting main component of application.
+                 *
+                 * @since [*next-version*]
+                 */
+                'eddbk_wizard_main_component_handler' => function (ContainerInterface $c) {
+                    return new MainComponentHandler(
+                        $c->get('eddbk_front_app_holder_template')
+                    );
+                },
+            ]);
+    }
+
+    /**
+     * Get content of template file.
+     * 
+     * @since [*next-version*]
+     * 
+     * @param string $templateFile Template file for reading.
+     * 
+     * @return string Content of given template file.
+     */
+    protected function _openTemplate($templateFile)
```

Comment:
The name of the method is confusing: it has nothing to do with templates. I think it should either retrieve template instances, or at least deal with files relative to the template directory (by the way, don't forget asset path mapping Puli style), or be renamed.